Modules/LearningSequence/classes/Player/class.ilKioskPageRenderer.php

```diff
@@ -5,21 +5,24 @@
 use ILIAS\UI\Renderer;
 use ILIAS\UI\Component\Component;
 use ILIAS\UI\Component\Listing\Workflow\Workflow;
+use ILIAS\GlobalScreen\Scope\Layout\LayoutServices;
 
 /**
  * Class ilKioskPageRenderer
  */
 class ilKioskPageRenderer
 {
 	public function __construct(
-		ilGlobalTemplate $il_global_template,
+		ilGlobalPageTemplate $il_global_template,
+		LayoutServices $layout_service,
```

Comment:
Let's try to pin down our real dependency. As far as I understand, we are not dependening on the complete `LayoutServices` here, but instead really only want two have two lists of files (JS, CSS) that we will include in the output. We could either demand them in the constructor or even just be open to receive them via a getter. I have not weighed pros and cons, though. This will make the class a lot easier to reuse (e.g. to test it), since the user (probably us...) won't need to give a complete `LayoutServices`.